call-home/src/collector/openapi/Cargo.toml

```diff
@@ -0,0 +1,66 @@
+[package]
```

Comment:
might be worth considering moving openapi to the root of the repo, in case other crates want to use it.

Reply:
In that case, should I also move mayastor-control-plane submodule to the root ?

Reply:
Yeah, or maybe a dependencies directory in the root? And maybe call it `control-plane` rather than `mayastor-control-plane`.

Reply:
done